### PR TITLE
Set up

### PR DESCRIPTION
**Pull request by aepaysinger**

### CODE REVIEW

File: weather/app/main.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from flask import Flask, escape, request
```
Comment (W-Ely) [inline weather/app/main.py:1]: `escape` is unused, add it once it's needed.

File: weather/app/main.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello():
+    name = request.args.get("name", "World")
+    return f"Hello, {escape(name)}!"
```
Comment (aepaysinger) [inline weather/app/main.py:10]: i use escape here

**Review by W-Ely — APPROVED** [review]
I just missed the escape usage. Do you know why you need it?

**Comment by aepaysinger** [comment]
i guess i don't need it here. but if i had & < > ' " i would. it converts those characters to HTML safe sequence. should i take it out for now?

**Comment by W-Ely** [comment]
> i guess i don't need it here. but if i had & < > ' " i would.

Where does `name` come from?  How do you know what it might contain?

**Comment by aepaysinger** [comment]
is name the value in the request object?

**Comment by W-Ely** [comment]
> is name the value in the request object?

name is not a value.

What does this do :
> name = request.args.get("name", "World")

**Comment by aepaysinger** [comment]
i meant key not value

**Comment by W-Ely** [comment]
Yup, question still stands though
What does this do `name = request.args.get("name", "World")`

**Comment by aepaysinger** [comment]
that sets name to "world"?

**Comment by W-Ely** [comment]
What is a `request` and where does the `args` property come from? 

Args is a dict like thing, so maybe the variable `name` under some conditions would be set to `World`, when would it not be?

Can you show me what a `curl` call would look like that sets it to `foobar` instead looks like?

**Comment by aepaysinger** [comment]
would it be something like curl - d 'name=foobar' https://localhost:5000 ??

**Comment by W-Ely** [comment]
> would it be something like curl - d 'name=foobar' https://localhost:5000/ ??

Did that work?

**Pull request merged**